onebusaway-android/src/main/java/org/onebusaway/android/report/ui/SimpleArrivalListFragment.java

```diff
@@ -81,10 +82,14 @@ public static void show(AppCompatActivity activity, Integer containerViewId,
         fragment.setArguments(FragmentUtils.getIntentArgs(intent));
         fragment.setCallback(callback);
 
-        FragmentTransaction ft = fm.beginTransaction();
-        ft.replace(containerViewId, fragment, TAG);
-        ft.addToBackStack(null);
-        ft.commit();
+        try {
+            FragmentTransaction ft = fm.beginTransaction();
+            ft.replace(containerViewId, fragment, TAG);
+            ft.addToBackStack(null);
+            ft.commit();
+        } catch (IllegalStateException e) {
+            Log.v("ArrivalListFragment", "Cannot show SimpleArrivalListFragment");
```

Comment:
Actually at first I did in that way, but it is giving an error because `SimpleArrivalListFragment` is longer than 23 chars:

![screen shot 2016-06-09 at 3 33 14 pm](https://cloud.githubusercontent.com/assets/2777974/15943590/b004cac2-2e57-11e6-95c0-94a533e3a741.png)